src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(double radius) {
+        setColor(Color.WHITE.name());
+        this.radius = radius;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: circle, area: " + obtainTheArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
+    }
+
+    @Override
+    public double obtainTheArea() {
+        return Math.PI * radius * radius;
```

Comment:
The method name 'obtainTheArea' could be simplified to 'getArea' to follow common Java naming conventions and to adhere to the single responsibility principle, as the method is responsible for obtaining the area of the figure.